### PR TITLE
[json rpc] update get-metadata specs

### PR DESCRIPTION
**Pull request by christinacelee**
## Motivation

Update JSON RPC specs for `get_metadata` endpoint

Addresses https://github.com/libra/libra/issues/4322

### CODE REVIEW

File: json-rpc/json-rpc-spec.md [hunk 1]
```diff
@@ -406,16 +406,33 @@ curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","metho
 
 **Description**
 
-Get the current blockchain metadata (e.g., current state of a Libra full node). All Read operations can be batched with get_metadata rpc to obtain synced metadata information with the request.
+Get the blockchain metadata (e.g., state of a Libra full node). All Read operations can be batched with get_metadata rpc to obtain synced metadata information with the request.
 
 You can use this endpoint to verify liveness / status of nodes in the network:
 
-get_metadata returns the latest transaction version and the block timestamp. If the timestamp or version is old (from the past), it means that the full node is not not up-to-date.
+get_metadata returns the transaction version and the block timestamp. If the timestamp or version is old (from the past), it means that the full node is not not up-to-date.
```
Comment (thefallentree) [inline json-rpc/json-rpc-spec.md:413]: double "not" here

File: json-rpc/json-rpc-spec.md [hunk 2]
```diff
@@ -406,16 +406,33 @@ curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","metho
 
 **Description**
 
-Get the current blockchain metadata (e.g., current state of a Libra full node). All Read operations can be batched with get_metadata rpc to obtain synced metadata information with the request.
+Get the blockchain metadata (e.g., state of a Libra full node). All Read operations can be batched with get_metadata rpc to obtain synced metadata information with the request.
```
Comment (thefallentree) [inline json-rpc/json-rpc-spec.md:409]: I will change to "state as known to the current full node "

**Comment by thefallentree** [comment]
I still think think is bad design, we should allow get_metatdata without param as before. 

I think we should create two use cases get_metadata and get_metatda_by_version  to distingutish

**Comment by davidiw** [comment]
Looks like a stale PR, closing out, feel free to re-open.

**Comment by christinacelee** [comment]
@xli @thefallentree let's get conclusion on this

**Comment by xli** [comment]
will have another PR to fix the issue, close this one

**Pull request closed without merging**